### PR TITLE
prevent accessible buffer from showing on click

### LINKED ISSUE #179020: Clicking Top Edge of Terminal break the Terminal
Type: <b>Bug</b>

1. Open Vscode Integrated Terminal
2. Click the top edge of terminal ( around tabs )
3. Can't enter command anymore.

Here's the screen recording:

https://user-images.githubusercontent.com/43804263/229553225-ea6340ac-492a-4efa-b0a8-d96f84648edf.mov

VS Code version: Code 1.77.0 (7f329fe6c66b0f86ae1574c2911b681ad5a45d63, 2023-03-29T09:55:48.214Z)
OS version: Darwin arm64 22.3.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|3, 3, 4|
|Memory (System)|8.00GB (0.05GB free)|
|Process Argv|--crash-reporter-id 2435e726-af01-43fd-aea5-91bbb347f71e|
|Screen Reader|no|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vstes627:30244334
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
nodejswelcome1cf:30587006
282f8724:30602487
pyind779:30671433
f6dab269:30613381
pythonsymbol12:30671437
showlangstatbar:30672706
vscodedisable:30660115
defaultazurewalk:30687958
pythonms35:30701012

```

</details>

### PR DESCRIPTION
**Pull request by meganrogge**
fix #179020

### CODE REVIEW

**Comment by meganrogge** [comment]
fyi @Tyriar

File: src/vs/workbench/contrib/terminal/browser/media/terminal.css [hunk 1]
```diff
@@ -616,3 +616,8 @@
 	z-index: 33;
 	background-color: var(--vscode-terminal-background, --vscode-panel-background);
 }
+
+.monaco-workbench .accessible-buffer {
+	pointer-events: none;
```
Comment (roblourens) [inline src/vs/workbench/contrib/terminal/browser/media/terminal.css:621]: Is this going to keep me from selecting/copying though?

**Review by roblourens — APPROVED** [review]

File: src/vs/workbench/contrib/terminal/browser/media/terminal.css (hunk 1 above)
Comment (roblourens) [reply src/vs/workbench/contrib/terminal/browser/media/terminal.css:621]: Seems like it should be `display: none` when not active

Comment (meganrogge) [reply src/vs/workbench/contrib/terminal/browser/media/terminal.css:621]: yeah when it's not active, you shouldn't be able to select or copy. see the css rule above that will allow you to do all of that

Comment (roblourens) [reply src/vs/workbench/contrib/terminal/browser/media/terminal.css:621]: Ah I get it

**Comment by davidje13** [comment]
As far as I can tell this fix ended up in [1.77.1](https://github.com/microsoft/vscode/releases/tag/1.77.1) (this went in 17 commits before the one used for that release), but I still see the issue in that version.

Maybe I don't understand how the releases are cut, but it looks like this might not be a full fix?

**Comment by meganrogge** [comment]
This fix did not make it in that build.

**Pull request merged**